stable_baselines3/common/logger.py

```diff
@@ -83,6 +105,9 @@ def write(self, key_values: Dict, key_excluded: Dict, step: int = 0) -> None:
             if excluded is not None and ("stdout" in excluded or "log" in excluded):
                 continue
 
+            if isinstance(value, Video):
+                raise FormatUnsupportedError(["stdout", "log"], str(value))
```

Comment:
ok, I think there was a misunderstanding there.
why would you use `str(value)` instead of something more readable like `"video"`?

Reply:
Ah I understand now :). Well my reasoning was that I could just use the string representation of the value as it might contain interesting information of the state of the value object, but it does make the error message quite verbose.

Reply:
ok, but then in case of the video, you will just check bytes, no?